### DISCUSSION TITLE
Need more posts

### DISCUSSION THREAD

**Discussion by seifalmotaz** [community/community#17473] [Other Feature Feedback, Questions, & Ideas]
I think the `For you` page is very good but needs to add more dynamic posts I need to see new posts every time I refresh the page.

And I have an idea that I would love to see on GitHub I want something like dev.to posts, by enabling the users and organizations to post articles in Github.

**Comment by ghost** [comment]
Seria bem interessante a implementação da sua ideia, nesta plataforma.

**Comment by seifalmotaz** [comment]
I think it will be great if GitHub add posts for users and organizations and articles for repos to maintain GitHub's identity.
